lib/msal-node/rollup.config.js

```diff
@@ -5,68 +5,49 @@
 
 import { nodeResolve } from "@rollup/plugin-node-resolve";
 import typescript from "@rollup/plugin-typescript";
+import { defineConfig } from "rollup";
 import pkg from "./package.json";
 
-const libraryHeader = `/*! ${pkg.name} v${pkg.version} ${new Date().toISOString().split("T")[0]} */`;
+const libraryHeader = `/*! ${pkg.name} v${pkg.version} ${
+    new Date().toISOString().split("T")[0]
+} */`;
 const useStrictHeader = "'use strict';";
 const fileHeader = `${libraryHeader}\n${useStrictHeader}`;
 
-export default [
-    {
-        // for cjs build
-        input: "src/index.ts",
-        output: {
+export default defineConfig({
+    input: "src/index.ts",
+    output: [
+        {
             dir: "dist",
             format: "cjs",
             entryFileNames: "[name].cjs",
             preserveModules: true,
             preserveModulesRoot: "src",
```

Comment:
seeing as the only valid entry-point for the module is `index.ts`, I don't understand why these are used